openassessment/xblock/peer_assessment_mixin.py

```diff
@@ -147,6 +153,15 @@ def render_peer_assessment(self, data, suffix=''):
 
         return self.render_assessment(path, context_dict)
 
+    @XBlock.handler
```

Comment:
Let's please avoid the core change, one possible way is to write a mixin in our application and inherit that mixin from this one implement this particular handler in that mixin and use new mixin everywhere.

Reply:
created a new mixin in our new application and removed the code from this core file